documentation/writing-policies.md

```diff
@@ -131,6 +131,24 @@ Operators supported:
 - Equal
 - NotEqual
 
+# Background processing
+Kyverno applies policies in foreground and background mode.
+- `foreground`: leverages admission control webhooks to intercept the resource api-request and apply policy on it.
```

Comment:
changes:
-  `to intercept, and apply policies on, API requests for resource changes`